snuba/settings/__init__.py

```diff
@@ -144,6 +144,11 @@
 
 # Processor/Writer Options
 
+# Mapping of (logical topic names, slice id) pairs to physical topic names
+# This is for Kafka topics that are associated
+# with multiple slices
+SLICED_KAFKA_TOPICS: Mapping[Tuple[str, int], str] = {}
```

Comment:
Could you add an example usage of this setting in the partitioning docs?

Reply:
Agree with this. Examples are much easier to parse than the long descriptions. The generic metrics use case would be a very good example to write in the comment since that is the first one we are gonna slice.

Reply:
Yep, I'm on it now! I was waiting for discussions and to see if any other config would be changed before I write up in docs and add examples etc